cloudos_cli/__main__.py

```diff
@@ -1783,15 +1799,17 @@ def cromwell_restart(ctx,
 @click.version_option()
 @click.option('-k',
               '--apikey',
-              help='Your CloudOS API key.')
+              help='Your CloudOS API key.',
+              required=True)
```

Comment:
This also will lead to issues, because in Cromwell you could use either apikey or cromwell-token